modules/ROOT/pages/dataweave-streaming.adoc

```diff
@@ -0,0 +1,397 @@
+= Streaming in DataWeave
+
+DataWeave supports end-to-end streaming through a flow in a Mule application. Streaming speeds the processing of large documents without overloading memory.
+
+DataWeave processes streamed data as its bytes arrive instead of scanning the entire document to index it. DataWeave can also pass streamed output data directly to a message processor without saving it to the disk. This behavior enables DataWeave and Mule to process data faster and consume fewer resources than the default processes for reading and writing data.
```

Comment:
I would say `DataWeave can also pass streamed output data directly to a message processor without saving it to the disk by enabling deferred mode`

Reply:
When in deferred mode, DataWeave can also pass streamed output data directly to a message processor without saving it to the disk.

Reply:
👍